stable/minecraft/README.md

```diff
@@ -0,0 +1,65 @@
+# Minecraft
+
+[Minecraft](https://minecraft.net/en/) Minecraft is a game about placing blocks and going on adventures.
+
+## Introduction
+
+This chart creates a single Minecraft Pod, plus Services for the Minecraft server and RCON.
+
+## Prerequisites
+
+- 512 MB of RAM
+- Kubernetes 1.4+ with Beta APIs enabled
+- PV provisioner support in the underlying infrastructure
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`, read the [Minecraft EULA](https://account.mojang.com/documents/minecraft_eula) run:
+
+```bash
+$ helm install --name my-release \
+    --set minecraftServer.eula=true stable/minecraft
+```
+
+This command deploys a Minecraft dedicated server with sensible defaults.
+
+> **Tip**: List all releases using `helm list`
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```bash
+$ helm delete my-release
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Configuration
+
+Refer to `values.yaml` for the full run-down on defaults. These are a mixture of Kubernetes and Minecraft-related directives that map to environment variables in the [itzg/minecraft-server](https://hub.docker.com/r/itzg/minecraft-server/) Docker image.
```

Comment:
It would be good to add a link to the values.yaml - https://github.com/kubernetes/charts/blob/master/stable/minecraft/values.yaml